src/prompts/alert/banner.scss

```diff
@@ -0,0 +1,70 @@
+.kumulos-banner-container {
+    display: flex;
+    flex-direction: row;
+
+    position: fixed;
+    left: 0;
+
+    width: 100%;
+    height: 80px;
+
+    padding: 10px;
+
+    border-color: #fff;
```

Comment:
Do we actually declare borders / is this necessary?

Reply:
Not really, can chop this!